From 37a145058e17e464398b5ac3ab8576115046a366 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Mon, 2 Jul 2012 21:13:34 +0800
Subject: [PATCH 018/602] ENGR00215569 arm2: remove the bad backsplash

The last backsplash in PRODUCT_COPY_FILES makes
the DEVICE_PACKAGE_OVERLAYS not defined.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/arm2/ProductCommonArm2.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/imx6/arm2/ProductCommonArm2.mk b/imx6/arm2/ProductCommonArm2.mk
index ad257d7..5b050cc 100644
--- a/imx6/arm2/ProductCommonArm2.mk
+++ b/imx6/arm2/ProductCommonArm2.mk
@@ -1,7 +1,5 @@
 PRODUCT_COPY_FILES += \
 	device/fsl/imx6/arm2/required_hardware.xml:system/etc/permissions/required_hardware.xml \
-    	device/fsl/imx6/arm2/vold.fstab:system/etc/vold.fstab \
-
-# GPU files
+	device/fsl/imx6/arm2/vold.fstab:system/etc/vold.fstab
 
 DEVICE_PACKAGE_OVERLAYS := device/fsl/imx6/arm2/overlay
-- 
1.8.0

